### PR TITLE
Add markerfacealt to pass-through arguments for error bar lines

### PR DESCRIPTION
**Pull request by wqian94**
## PR Summary

Add `markerfacealt` as a line kwarg that should be passed through when making error bars. See issue #23375 

## PR Checklist

**Tests and Styling**
- [✔️] Has pytest style unit tests (and `pytest` passes).
- [✔️] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [N/A] New features are documented, with examples if plot related.
- [N/A] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [✔️] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [✔️] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -3367,7 +3367,25 @@ def errorbar(self, x, y, yerr=None, xerr=None,
...
+                - *linestyle*
+                - *marker*
+                - *markeredgecolor*
+                - *markeredgewidth*
+                - *markerfacecolor*
+                - *markerfacecoloralt*
+                - *markersize*
+                - *markevery*
+                - *solid_capstyle*
+                - *solid_joinstyle*
```
Comment (timhoffm) [inline lib/matplotlib/axes/_axes.py:3386]:
```suggestion
            - *dashes*
            - *dash_capstyle*
            - *dash_joinstyle*
            - *drawstyle*
            - *fillstyle*
            - *linestyle*
            - *marker*
            - *markeredgecolor*
            - *markeredgewidth*
            - *markerfacecolor*
            - *markerfacecoloralt*
            - *markersize*
            - *markevery*
            - *solid_capstyle*
            - *solid_joinstyle*
```

(ReStructuredText formatting pecularities.)

**Review by timhoffm — APPROVED** [review]
Modulo a formatting issue.

**Review by oscargus — APPROVED** [review]

**Comment by oscargus** [comment]
Thanks and congratulations on your first merged PR @wqian94 ! Hope to see you around!

**Comment by wqian94** [comment]
Thanks!

**Pull request merged**